docs/project-setup/filesystem-sync.md

```diff
@@ -44,13 +25,13 @@ volumes:
 ```
 
 !!! important "Note about volume name"
-    If you optionally specified a volume name to the `project sync:start` command or configured a name in your Outrigger 
+    If you optionally specified a volume name to the `project sync:start` command or configured a name in your Outrigger
```

Comment:
In the example above, naming the volume "projectname-sync:" will help make it more clear that one shouldn't use the literal value "project-sync" there.